geomesa-spark/geomesa-spark-sql/pom.xml

```diff
@@ -61,9 +61,14 @@
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-cqengine_${scala.binary.version}</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-cqengine-datastore_${scala.binary.version}</artifactId>
             <version>${project.version}</version>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
@jnh5y Same build error when changing this to provided.

What is the downside to just adding the GeoCQEngineDataStore dependency to the leaflet module with scope 'provided'? We did this for HBase Spark Runtime.